### PR TITLE
docs: Add Web UI documentation and update FAQ / feature retrieval docs

### PR DESCRIPTION
**Pull request by adchia**
Adding documentation for the Feast Web UI, updates the FAQ with some common answers, and also adds more description to feature services as part of the feature retrieval page

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by achals — APPROVED** [review]
/lgtm

File: docs/getting-started/faq.md [hunk 1]
```diff
@@ -77,7 +96,7 @@ Please follow the instructions [here](../how-to-guides/adding-support-for-a-new-
 
 ### Can the same storage engine be used for both the offline and online store?
 
-Yes. For example, the Postgres [connector](https://github.com/nossrannug/feast-postgres) can be used as both an offline and online store.
+Yes. For example, the Postgres connector can be used as both an offline and online store.
```
Comment (achals) [inline docs/getting-started/faq.md:99]: Wanna point out that postgres can also be used to store the registry?

File: sdk/python/feast/ui/yarn.lock [hunk 2]
```diff
@@ -1345,10 +1345,10 @@
     minimatch "^3.1.2"
     strip-json-comments "^3.1.1"
 
-"@feast-dev/feast-ui@^0.20.2":
-  version "0.20.2"
-  resolved "https://registry.yarnpkg.com/@feast-dev/feast-ui/-/feast-ui-0.20.2.tgz#7af94f99f995bf6c0625ca504307fa2630f509e9"
-  integrity sha512-K5O5uTEkTtW3uuoI49YsZAutdv38T/psWygNcNpQ9UWMrlQCfHjMU0gTSPJpakVVIFoWRvn8C3Q4HN8tLB6ZAg==
+"@feast-dev/feast-ui@^0.20.3":
+  version "0.20.3"
```
Comment (achals) [inline sdk/python/feast/ui/yarn.lock:1349]: Why was this bumped manually?

Comment (adchia) [reply sdk/python/feast/ui/yarn.lock:1349]: was accidentally missed in a previous PR

File: docs/getting-started/faq.md (hunk 1 above)
Comment (adchia) [reply docs/getting-started/faq.md]: done, though doesn't look like we have any docs about that.

**Pull request merged**